app/build.gradle

```diff
@@ -80,6 +80,11 @@ dependencies {
     implementation 'androidx.preference:preference:1.2.0'
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
 
+    // QR code generation
+    implementation 'com.google.zxing:core:3.3.0'
+
+    implementation 'com.google.android.gms:play-services-maps:18.0.2'
+
```

Comment:
These lines can be removed but it could be cleanup work

Reply:
Already done with las commit, thanks!